Zend/tests/final_properties/access/property_assignment1.phpt

```diff
@@ -0,0 +1,104 @@
+--TEST--
+Test that final properties can't be mutated by assignment operators
```

Comment:
These tests might not be very useful because all of them boil down to simple assignments from the perspective of "write-once" properties